Makefile

```diff
@@ -0,0 +1,10 @@
+#this will execute the program 
+OBJ=obj
+all: 
+	mkdir -p $(OBJ) 
+	gcc Hello\ World.c -o $(OBJ)/hello-world
+clean: 
```

Comment:
add an empty line before this, same for other places.

Reply:
yes